modules/nodes-containers-downward-api-container-values-plugin.adoc

```diff
@@ -64,15 +66,18 @@ spec:
           fieldPath: metadata.annotations
         path: pod_annotations
   restartPolicy: Never
+# ...
 ----
 
-. Create the pod from the `volume-pod.yaml` file:
+.. Create the pod from the `volume-pod.yaml` file:
 +
 [source,terminal]
 ----
 $ oc create -f volume-pod.yaml
 ----
 
+.Verification
+
```

Comment:
I don't think this should have a number, if it's the only verification criteria. See RH supplementary style guide: https://redhat-documentation.github.io/supplementary-style-guide/#single-step-procedures